views/jira-redirect.hbs

```diff
@@ -10,6 +10,8 @@
 
 <body>
    Redirecting to your GitHub Enterprise Server Instance
+
+   <input type="hidden" id="baseUrl" value="{{baseUrl}}">
```

Comment:
Instead of passing via hidden textfield, pass it via query paramter.

Reply:
This `baseUrl` is passed from `GithubRedirectGet` as an `option`  inside `res.render`. How can we pass this value from the `backend` to the frontend `URL`?

Reply:
Why to pass it from backend in this case? Grab it directly from the URL on the frontend.

Reply:
It can't be grabbed directly from the frontend because the `queryParameter` is not present when getting to this view. The user gets to this view after getting redirected, during which we don't have the `baseUrl`.
 
From the frontend, the user is firstly redirected to `"/session/github/redirect?baseUrl=" + gitHubServerBaseUrl`, which takes the user to `GithubRedirectGet`. From there, we now pass this `queryParameter` to the next view `jira-redirect.hbs`. Without passing it like this, there is no medium to pass or get the `baseUrl` into the required view.

Reply:
User is redirected from `/session/github/redirect?baseUrl=<host>` to `/github/redirect?baseUrl=<host>`. The baseUrl is present on both views.

In `jira-redirect.js`, you can get baseURL using -
```
const params = new URLSearchParams(window.location.search.substring(1));
const gheHost = params.get("baseUrl");
```

Reply:
Updated